presto-main/src/test/java/io/prestosql/cost/TestFilterStatsCalculator.java

```diff
@@ -294,6 +297,9 @@ public void testAndStats()
 
         assertExpression("'a' IN ('b', 'c') AND unknownRange = 3e0")
                 .outputRowsCount(0);
+
+        assertExpression("CAST(NULL AS boolean) AND CAST(NULL AS boolean)").equalTo(zeroStatistics);
```

Comment:
maybe add a comment why these tests are special

Reply:
They aren't very special. Just a case that wasn't there.